monai/handlers/tensorboard_handlers.py

```diff
@@ -194,6 +196,9 @@ def __init__(
             summary_writer (SummaryWriter): user can specify TensorBoard SummaryWriter,
                 default to create a new writer.
             log_dir (str): if using default SummaryWriter, write logs to this directory, default is `./runs`.
+            interval (int): plot image every N epochs or every N iterations, default is 1.
```

Comment:
plot image -> plot content from `engine.state`

Reply:
updated in the latest patch.
Thanks.